### DISCUSSION TITLE
Publish your own rendertheme

### DISCUSSION THREAD

**Discussion by troNpo** [mapsforge/vtm#1080] [Q&A] [answered]
Hello, I have modified the default rendertheme of the VTM repository, I would like to make it public for other colleagues, what conditions or requirements, do I have to meet
Where can I find out.

**Comment by devemux86** [comment, marked as answer]
@troNpo very nice!

Mapsforge and VTM have [LGPL license](https://github.com/mapsforge/vtm/blob/master/COPYING.LESSER).

The only requirement is that any modifications to the code or their resources must be published under the same license.
